qiskit_nature/circuit/library/ansatzes/utils/fermionic_excitation_generator.py

```diff
@@ -128,6 +124,29 @@ def generate_fermionic_excitations(
         The list of excitations encoded as tuples of tuples. Each tuple in the list is a pair. The
         first tuple contains the occupied spin orbital indices whereas the second one contains the
         indices of the unoccupied spin orbitals.
+
+    Examples:
+        Generate excitations with basic inputs.
+
+        >>> from qiskit_nature.circuit.library.ansatzes.utils.fermionic_excitation_generator import (\
+        generate_fermionic_excitations)
+        >>> excitations = generate_fermionic_excitations(num_excitations = 1, num_spin_orbitals = 4,\
+        num_particles = (1,1))
+        >>> print(excitations)
+        [((0,), (1,)), ((2,), (3,))]
+
+        Generate excitations limiting the maximum number of excitations per spin species.
+
+        >>> excitations = generate_fermionic_excitations(1, 4, (1, 1), max_spin_excitation=1)
+        >>> print(excitations)
+        [((0,), (1,)), ((2,), (3,))]
+
+        Generate generalized excitations.
+
+        >>> excitations = generate_fermionic_excitations(1, 4, (1, 1), generalized=True)
+        >>> print(excitations)
+        [((0,), (1,)), ((2,), (3,))]
```

Comment:
Although these are all different examples, showing different parameter usage, they all end up as the same list being printed. What might be nice is an example where the parameter being shown makes a difference to the outcome.

For parameters in the function calls there should be no space around = for the style used. This is the case in the latter 2 examples but not the first.

As Python will print the return of a function when invoked interactively I think the examples can avoid the print and be more like the following.
```
        >>> generate_fermionic_excitations(1, 4, (1, 1), generalized=True)
        [((0,), (1,)), ((2,), (3,))]
```

Reply:
Thanks Steve - I've made the changes 😄